content/blogposts/blog-post-starter-pack.md

```diff
@@ -0,0 +1,264 @@
+---
```

Comment:
Removing all these empty "Blogy McBlogerson" blogposts, no one knew about these blogposts. They probably outdate most of marketing and don't need to be in prod.